### ISSUE TITLE
Improve submenu appearance 

### ISSUE THREAD

**Issue by hectorsector** [github/training-kit#498]
## Overview
**TL;DR**
The submenus (currently in use in [Git Out of Trouble](https://services.github.com/on-demand/git-trouble/)) don't appear to be submenus. We need to alter the way this renders so it's easier to see that these are submenu items. This came up in https://github.com/github/training-kit/pull/478#issuecomment-290496285.

#### Screenshots

![](https://cloud.githubusercontent.com/assets/9950121/24519271/9b55c0c6-1552-11e7-8976-ada951ec91ee.png)

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by hectorsector** [comment]
@JasonEtco did this and we now have much prettier submenus! 🎉 

<img width="638" alt="screen shot 2017-08-25 at 3 59 53 pm" src="https://user-images.githubusercontent.com/16547949/29730527-77e6ab3e-89ae-11e7-86ce-dc4a25e6db50.png">
